### PR TITLE
(issue 871): implemented the allclose() method

### PR DESCRIPTION
**Pull request by alixdamman**

### CODE REVIEW

**Comment by alixdamman** [comment]
I wonder who will use `allclose` and `isclose` except you. Hard to say. 
OK to remove `isclose` from the PR.

**Review by gdementen — COMMENTED** [review]
> I wonder who will use `allclose` and `isclose` except you. Hard to say.
> OK to remove `isclose` from the PR.

🤷‍♂️ ... you? 😄

**Comment by gdementen** [comment]
BTW, this LGTM

File: larray/core/array.py [hunk 1]
```diff
@@ -5773,6 +5775,80 @@ def equals(self, other, rtol=0, atol=0, nans_equal=False, check_axes=False):
...
+        Returns
+        -------
+        bool
+            Returns True if the two arrays are equal within the given tolerance; False otherwise.
+
+        See Also
+        --------
+        Array.equals, Array.isclose
+
+
```
[the remark this answers is not in the document]
Comment (alixdamman) [reply larray/core/array.py:5816]: and remove `Array.isclose` above 😄

[the remark this answers is not in the document]
Comment (gdementen) [reply larray/core/array.py:5816]: 😄 That's what I get for forgetting to submit my reviews (happens often!) and then submit an "old" code comment along with a new general comment...

**Pull request merged**